types/node/buffer.buffer.d.ts

```diff
@@ -135,7 +135,7 @@ declare module "buffer" {
              * @param list List of `Buffer` or {@link Uint8Array} instances to concatenate.
              * @param totalLength Total length of the `Buffer` instances in `list` when concatenated.
              */
-            concat(list: readonly Uint8Array[], totalLength?: number): Buffer<ArrayBuffer>;
+            concat(list: readonly Buffer[] | readonly Uint8Array[], totalLength?: number): Buffer<ArrayBuffer>;
```

Comment:
`Buffer` is a superset of `Uint8Array`; this is unnecessary.

Reply:
Ah yes, I thought this was a bug with Node.js, but it is a bug with `bun.sh`